doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst

```diff
@@ -245,3 +254,118 @@ The following samples allow you to test if the buttons (or switches) and LEDs on
+| D9               | DATA3                 | QSPI data line 3                 |
++------------------+-----------------------+----------------------------------+
+| D10              | SS                    | Slave select                     |
++------------------+-----------------------+----------------------------------+
+| D11              | MISO/DATA1            | QSPI/SPI Data line 1/ Slave Out  |
++------------------+-----------------------+----------------------------------+
+| D12              | MOSI/DATA0            | QSPI/SPI Data line 0/ Slave In   |
++------------------+-----------------------+----------------------------------+
+| D13              | CLK                   | QSPI/SPI Clock                   |
++------------------+-----------------------+----------------------------------+
+| GND              | GND                   | Ground                           |
++------------------+-----------------------+----------------------------------+
+| AREF             | N.C.                  | Not used                         |
++------------------+-----------------------+----------------------------------+
+| SDA              | N.C.                  | Not used                         |
++------------------+-----------------------+----------------------------------+
+| SCL              | N.C.                  | Not used                         |
++------------------+-----------------------+----------------------------------+
+
+Programming
+-----------
+
+To add support for the nRF7002 EK on an application running on a compatible host development board, the ``SHIELD`` setting must be specified.
+
+The nRF7002 EK features the nRF7002 companion IC.
+In addition, the shield may be used to emulate the nRF7001 and nRF7000 companion IC variants
+
+To add support for the nRF7002 EK and the nRF7002 IC, set ``-DSHIELD=nrf7002ek`` when you invoke ``west build`` or ``cmake`` in your |NCS| application.
+To emulate support for the nRF7001 or nRF7000 ICs, specify ``-DSHIELD=nrf7002ek_nrf7001`` or ``-DSHIELD=nrf7002ek_nrf7000``, respectively.
+
+Alternatively, add the shield in the project's :file:`CMakeLists.txt` file, specifying the below settings, depending on which IC is to be used:
+
+.. code-block:: console
+
+   set(SHIELD nrf7002ek)
+
+.. code-block:: console
+
+   set(SHIELD nrf7002ek_nrf7001)
+
+.. code-block:: console
+
+   set(SHIELD nrf7002ek_nrf7000)
+
+To build with the |nRFVSC|, specify ``-DSHIELD=nrf7002ek`` in the **Extra Cmake arguments** field.
+See :ref:`cmake_options`.
+
+To build for the nRF7002 EK and the nRF7002 IC with nRF5340 DK, use the ``nrf5340dk_nrf5340_cpuapp`` build target with the CMake ``SHIELD`` variable set to ``nrf7002ek``.
+For example, you can use the following command when building on the command line:
+
+.. code-block:: console
+
+   west build -b nrf5340dk_nrf5340_cpuapp -- -DSHIELD=nrf7002ek
+
+References
+----------
+
+* `nRF70 Series product page`_
+* `nRF70 Series`_
```

Comment:
Can we include "infocenter" here? Somehow in needs to differ from the description above since one is pointing to nordicsemi.com and the latter to the infocenter...

Reply:
Added "hardware documentation".